ouroboros-consensus/src/Ouroboros/Consensus/Mempool/TxSeq.hs

```diff
@@ -152,33 +155,9 @@ splitAfterTicketNo (TxSeq txs) n =
     case FingerTree.split (\m -> mMaxTicket m > n) txs of
       (l, r) -> (TxSeq l, TxSeq r)
 
--- | Convert a 'TxSeq' to the type which is expected to be returned from the
--- 'Mempool' API.
+-- | Convert a 'TxSeq' to a list of pairs of transactions and their
+-- associated 'TicketNo's.
 fromTxSeq :: TxSeq tx -> [(tx, TicketNo)]
 fromTxSeq (TxSeq ftree) = fmap
   (\(TxTicket tx tn) -> (tx, tn))
   (Foldable.toList $ ftree)
-
--- | Append a @tx@ to the back of a 'TxSeq'.
```

Comment:
Nice that we can get rid of this, I was hoping we could (and use `:>` instead) :grin:

Reply:
For sure!